Pan os change commands names (#17632)
demisto/content: Packs/PAN-OS/ReleaseNotes/1_9_0.md

#### Integrations
##### Palo Alto Networks PAN-OS

> - Deprecated the commands starting with **panorama** prefix.
> - Changed the commands that start with **panorama** prefix to use **pan-os** prefix instead.

Reviewer: ```suggestion
- The commands with the **panorama** prefix are deprecated. Use the commands with the **pan-os** prefix instead.
```

#### Integrations
##### Palo Alto Networks PAN-OS
- The commands with the **panorama** prefix are deprecated. Use the commands with the **pan-os** prefix instead.

#### Playbooks
##### NetOps - Firewall Version and Content Upgrade

> - The commands with the **panorama** prefix are deprecated. The playbook uses the commands with the **pan-os** prefix instead in its tasks.

Reviewer: change all playbook release notes to Maintenance and stability enhancements.
